ui/cypress/e2e/variables.test.ts

```diff
@@ -203,6 +203,23 @@ describe('Variables', () => {
     )
   })
 
+  it('Can upload variable file', () => {
+    cy.getByTestID('add-resource-dropdown--button').click()
```

Comment:
We probably want to add this to an existing test so as to reduce the amount tests we're running, since each test that is booted up takes more time

Reply:
alright ill combine it with the other test above it